Secure Score/Diagnostic Logs/Log Analytics/README.md

```diff
@@ -0,0 +1,32 @@
+# Enable diagnostic logging automatically on your resources
```

Comment:
Is the same readme file will represent all 3 templates? (1 readme file with 3 templates in the same directory?)
if not, please remeove the irrelevant parts (storae, EH)